src/main/scala/firrtl/passes/AnnotateMemMacros.scala

```diff
@@ -21,35 +21,35 @@ case class AppendableInfo(fields: Map[String, Any]) extends Info {
 }
 
 object AnalysisUtils {
-
-  def getConnects(m: Module) = {
-    val connects = mutable.HashMap[String, Expression]()
-    def getConnects(s: Statement): Statement = {
-      s map getConnects match {
+  type Connects = collection.mutable.HashMap[String, Expression]
```

Comment:
From software engineering point of view, it's evil to return mutable data structures. However, I noticed that converting HashMap to immutable.Map can be slow.(It may not be true though)

We can just change Connects to immutable.Map if we think it should be.